woocommerce/woocommerce
# [Product Block Editor]: add CashRegister image component

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes # .



### How to test the changes in this Pull Request:

Let's use Storybook

1. Update, just in case:

```cli
pnpm install
````

2. Run the Storybook app

```cli
pnpm --filter=storybook run storybook
```

3. Go to the ShippingBags story:  http://localhost:6007/?path=/story/product-editor-images-cashregister--default
4. Confirm you see the CashRegister image component 
5. You may like to play with the addons panel

<img width="881" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/e0f3ee2e-91bb-49c5-a350-bff8b3c26915">





Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
